jenkins/shared-library/vars/publish.groovy

```diff
@@ -30,7 +30,7 @@ void dockerPublisher(Map config, String phase) {
 	String version = readPackageVersion()
 	String imageVersionName = "${config.dockerImageName}:${version}"
 
-	Object imageName = docker.build(imageVersionName)
+	Object imageName = docker.build(imageVersionName, config.dockerBuildArgs)
```

Comment:
``config.dockerBuildArgs`` will be null if not set in the pipeline. not sure how this will be handle by docker

Reply:
I thought it should be handled but just found the implementation(see the img below), updated accordingly, should be fine with the default '.' now.
![image](https://user-images.githubusercontent.com/6256269/208648358-cf13444c-72a5-47be-a87b-1907c7fefc2d.png)